src/natcap/invest/hra.py

```diff
@@ -222,321 +222,327 @@
-                    fields=[
-                        spec.IntegerOutput(
-                            id="Risk Score",
-                            about=gettext(
-                                "Habitat risk from all stressors where 0 = No Risk, 1 ="
-                                " Low Risk, 2 = Medium Risk, and 3 = High Risk."
-                            )
-                        )
-                    ]
-                ),
-                spec.VectorOutput(
-                    id="RECLASS_RISK_Ecosystem.geojson",
-                    about=gettext(
-                        "Map of ecosystem risk visualized in gradient color from white to"
-                        " red on a map."
-                    ),
-                    geometry_types={"POLYGON"},
-                    fields=[
-                        spec.IntegerOutput(
-                            id="Risk Score",
-                            about=gettext(
-                                "Ecosystem risk from all stressors where 0 = No Risk, 1 ="
-                                " Low Risk, 2 = Medium Risk, and 3 = High Risk."
-                            )
-                        )
-                    ]
-                ),
-                spec.VectorOutput(
-                    id="STRESSOR_[STRESSOR].geojson",
-                    about=gettext("Map of stressor extent visualized in orange color."),
-                    geometry_types={"POLYGON"},
-                    fields=[]
-                ),
-                spec.CSVOutput(
-                    id="SUMMARY_STATISTICS.csv",
-                    about=gettext(
-                        "This is the same file from one in the Output Folder. It is"
-                        " copied here so users can just upload the visualization outputs"
-                        " folder to the HRA web application, with all the files in one"
-                        " place."
-                    ),
-                    columns=[],
-                    index_col=None
-                )
-            ]
+        spec.SingleBandRasterOutput(
+            id="TOTAL_RISK_Ecosystem",
+            path="outputs/TOTAL_RISK_Ecosystem.tif",
+            about=gettext(
+                "Sum of habitat cumulative risk scores divided by the number of"
+                " habitats occurring in each cell."
+            ),
+            data_type=float,
+            units=u.none
         ),
-        spec.DirectoryOutput(
-            id="intermediate_outputs",
-            about=None,
-            contents=[
-                spec.SingleBandRasterOutput(
-                    id="aligned_[HABITAT/STRESSOR/CRITERIA].tif",
-                    about=gettext(
-                        "Copy of the input, aligned to the same projection and extent"
-                    ),
-                    data_type=float,
-                    units=u.none
+        spec.SingleBandRasterOutput(
+            id="RECLASS_RISK_[HABITAT]",
```

Comment:
I'm not sure why these output file names were written in all caps, but I think it would make sense to standardize all ids to lowercase.